src/components/GlobalStyles/GlobalStyles.js

```diff
@@ -105,74 +180,69 @@ const GlobalStyle = createGlobalStyle`
   h4, .h4,
   h5, .h5,
   h6, .h6 {
-    font-family: 'Lato', sans-serif;
+    font-family: var(--font-stack-heading);
     font-weight: 700;
     margin-top: 0;
   }
   
   h1, .h1 {
-    font-size: 4.0rem;
-    line-height: 1.15em;
+    font-size: var(--h1-sm);
```

Comment:
Have you thought about looking into [clamp for font sizes](https://www.smashingmagazine.com/2022/01/modern-fluid-typography-css-clamp/)?